src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -214,7 +215,7 @@ private static ClaimsIdentity CreateIdentity(TwoFactorAuthenticationInfo info)
             }
             var token = await UserManager.GenerateTwoFactorTokenAsync(user, provider);
             await UserManager.NotifyTwoFactorTokenAsync(user, provider, token);
-            return await LogResultAsync(true, user);
+            return await true.WithLoggingAsync(this, user);
```

Comment:
The token is always generated so there is not false result as such hence omitted that